tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/lock/LocksWithOnFailTest.java

```diff
@@ -57,6 +56,10 @@ public void testLockWithinLock() {
 
         assertEquals(returnsWithCheck.get(0), 100L);
         assertEquals(returnsWithCheck.get(1).toString(), "Error caught");
+    }
 
+    @Test
+    public void testOnFailLockWithinLockWithoutVariable() {
+        BRunUtil.invoke(compileResult, "onFailLockWithinLockWithoutVariable");
```

Comment:
Is there a difference in asserting the values here vs. within the test case itself? Maybe all `LocksWithOnFailTest`s could follow same structure for consistency.

Reply:
It is preferred that we do assertions in Ballerina itself for newly added tests.